### ISSUE TITLE
Support multiple gems per repo

### ISSUE THREAD

**Issue by namelessjon** [zapnap/rdocinfo#26]
Several projects (e.g. merb, datamapper) have one or more repositories which feature multiple gems in one git repository.  It would be cool if these could be documented as seperate gems on rdoc.info

**Comment by zapnap** [comment]
Right now these can be indexed (all together) by submitting a custom .document file as described on the site. 

If you'd rather see them automatically broken up and a hierarchical index presented, I'm not quite sure how we'd do that. If you have an idea, we'd be happy to listen (or feel free to fork the repo and experiment)!

**Comment by namelessjon** [comment]
I don't think it could be done automatically, no.  I don't expect omniscience from the generator ;)  Unless maybe it went looking for gemspec files, and assumed they were the roots of gems ...

Hmmm.  I'll experiment.
